metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -174,6 +175,14 @@ def __init__(self, attributes=None, statically_defined=False):
             except (NameError, ImportError):
                 pass
 
+        # parse the CPU, memory, disk, values from the KUBERNETES_RESOURCES environment variable (you would need to export the METAFLOW_KUBERNETES_RESOURCES environment variable with the desired values before running the flow)
+        # find the values in the format "cpu=1,memory=4096,disk=10240", if not set, the default values described above are used
+        if KUBERNETES_RESOURCES:
```

Comment:
thank you @saikonen , I did consider this and merely proceeded this way because of the way it was being passed via the command line arguments. I agree it is cleaner to separate, will fix now.